src/spi.rs

```diff
@@ -585,13 +505,13 @@ impl<SPI: Instance, PINS, TRANSFER_MODE, OPERATION> Spi<SPI, PINS, TRANSFER_MODE
 
     /// Return `true` if the TXE flag is set, i.e. new data to transmit
     /// can be written to the SPI.
-    pub fn is_txe(&self) -> bool {
+    pub fn is_tx_empty(&self) -> bool {
```

Comment:
What is the motivation for these renames?

Reply:
This is not principal.
Main motivation is compatibility with f1xx-hal:
https://github.com/stm32-rs/stm32f1xx-hal/blob/6c6c8c5154510b519fdf74207d5ff626accc0e39/src/spi.rs#L450

Reply:
Revert? Or make copy and deprecate?

Reply:
I hadn't thought of that, but copy and deprecate sounds better to me.

Reply:
done. See last commit